apps/src/code-studio/progressRedux.js

```diff
@@ -328,33 +328,60 @@ const peerReviewLevels = state => state.peerReviewStage.levels.map((level, index
 }));
 
 /**
+ * Determine whether the passed in level is our current level (i.e. in the dots
+ * in our header
+ * @returns {boolean}
+ */
+const isCurrentLevel = (state, level) => {
+  const currentLevelId = state.currentLevelId;
+  return !!currentLevelId &&
+    ((level.ids && level.ids.map(id => id.toString()).indexOf(currentLevelId) !== -1) ||
+    level.uid === currentLevelId);
+};
```

Comment:
This is the same as the logic used in ProgressDot. This does intentionally remove the logic around `stage_extras`. This was a feature recently added by Elijah behind a flag, that we've decided we're going to initially ignore and then redo the work for once we've got our redesigned bubbles (tracked in spec).